pulseapi/settings.py

```diff
@@ -51,10 +51,19 @@
     'test.example.com,localhost,network-pulse-api-staging.herokuapp.com,network-pulse-api-production.herokuapp.com'
 ).split(',')
 
+
+# Create a simple function to show Django Debug Toolbar on Review App
+def show_toolbar(request):
+    return request.user.is_staff
+
+
 # Adding support for Heroku review app
 if env('HEROKU_APP_NAME'):
     herokuReviewAppHost = env('HEROKU_APP_NAME') + '.herokuapp.com'
     ALLOWED_HOSTS.append(herokuReviewAppHost)
+    DEBUG_TOOLBAR_CONFIG = {
```

Comment:
does this override https://github.com/mozilla/network-pulse-api/blob/2616bdabd3f742f1625e436962daad93dcabbf58/pulseapi/settings.py#L113-L118?